### PR TITLE
Added extrahorns mod

### PR DESCRIPTION
**Pull request by DonutNinja**
2 new horns and bagpipes added

### CODE REVIEW

File: mods/extrahorns/depends.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+soundblocks
+default
+ wool
```
Comment (ClobberXD) [inline mods/extrahorns/depends.txt]: Please remove the extra space before `wool`

File: mods/extrahorns/init.lua [hunk 2]
```diff
@@ -0,0 +1,67 @@
+-- large horn
+minetest.register_craftitem ("extrahorns:largehorn", {
+    description     = "Large Horn"         ,
+    inventory_image = "large.png" ,
+    on_use          = function (itemstack, user)
+        minetest.sound_play ("HornLarge", {
+            pos               = user:getpos() ,
+            max_hear_distance = 450           ,
```
Comment (ClobberXD) [inline mods/extrahorns/init.lua]: No need to align commas. :)

File: mods/extrahorns/init.lua [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            gain              = 1.2             ,
+        })
+    end,
+    groups = {instrument=1}
+})
+
+minetest.register_craft ({
+    output = "extrahorns:warhorn",
+    recipe = {
+{ "extrahorns:largehorn" , ""           , ""              } ,
```
Comment (ClobberXD) [inline mods/extrahorns/init.lua]: Requires indentation. The two lines below, as well.

File: mods/extrahorns/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+This mod is an addon to soundblocks it will not work without that mod so here is the link to soundblocks
+https://forum.minetest.net/viewtopic.php?t=16619
+
+The sound used in the item "warhorn" was a modified version of the sound made by newagesoup on freesound and can be found here
```
Comment (ClobberXD) [inline mods/extrahorns/README.md]: ... *is* a modified version ...

File: mods/extrahorns/init.lua (hunk 3 above)
Comment (ClobberXD) [reply mods/extrahorns/init.lua]: Also, single spaces after the commas will suffice.

**Review by ClobberXD — COMMENTED** [review]
- A couple of code-style issues. Please also replace all the whitespaces by tab-spaces for consistency.
- The name of the sound files are inconsistent, and don't follow Minetest's naming system. e.g. The sound played by `extrahorns:largehorn` should be named `extrahorns_largehorn` (make sure it has a `.ogg` extension).

File: mods/extrahorns/depends.txt (hunk 1 above)
Comment (DonutNinja) [reply mods/extrahorns/depends.txt]: Fixed

File: mods/extrahorns/init.lua (hunk 2 above)
Comment (DonutNinja) [reply mods/extrahorns/init.lua]: Fixed

File: mods/extrahorns/init.lua (hunk 3 above)
Comment (DonutNinja) [reply mods/extrahorns/init.lua]: Fixed

File: mods/extrahorns/README.md (hunk 4 above)
Comment (DonutNinja) [reply mods/extrahorns/README.md]: Fixed

File: mods/extrahorns/init.lua [hunk 5]
```diff
@@ -22,46 +22,46 @@ minetest.register_craft ({
...
-    groups = {instrument=1}
 })
 
 minetest.register_craft ({
     output = "extrahorns:warhorn",
     recipe = {
-{ "extrahorns:largehorn" , ""           , ""              } ,
-{ ""           , "extrahorns:largehorn" , ""              } ,
-{ ""           , ""           , "extrahorns:largehorn" } ,
+		{ "extrahorns:largehorn" , "", ""} ,
```
Comment (ClobberXD) [inline mods/extrahorns/init.lua]: I still see spaces before the `,`

**Comment by ClobberXD** [comment]
You've added back only 3 sound files, maybe you missed the Dwarven one?

**Comment by DonutNinja** [comment]
> You've added back only 3 sound files, maybe you missed the Dwarven one?

Uhh no... originally i was going to use the Dwarven sound instead of the current Warhorn sound but copyright :/

File: mods/extrahorns/init.lua (hunk 5 above)
Comment (DonutNinja) [reply mods/extrahorns/init.lua]: i dont see that part of text :/

File: mods/extrahorns/init.lua [hunk 6]
```diff
@@ -37,9 +37,9 @@ minetest.register_craftitem ("extrahorns:warhorn", {
 minetest.register_craft ({
     output = "extrahorns:warhorn",
     recipe = {
-		{ "extrahorns:largehorn" , "", ""} ,
-		{ "", "extrahorns:largehorn" , ""} ,
-		{ "", "", "extrahorns:largehorn" } ,
+		{"extrahorns:largehorn","",""},
+		{"","extrahorns:largehorn",""},
+		{"","","extrahorns:largehorn"},
```
Comment (ClobberXD) [inline mods/extrahorns/init.lua]: You've deleted the spaces after commas as well. Please add them back. I only suggested removing the ones **before** `,`

Comment (DonutNinja) [reply mods/extrahorns/init.lua]: oh for

File: mods/extrahorns/init.lua [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+minetest.register_craftitem ("extrahorns:warhorn", {
+    description     = "War Horn" ,
+    inventory_image = "warhorn.png",
+    groups = {instrument=1},	
+    on_use = function (itemstack, user)
+        minetest.sound_play ("extrahorns_warhorn", {
+            pos               = user:getpos() ,
+            max_hear_distance = 550           ,
+            gain              = 1.2             ,
```
Comment (ClobberXD) [inline mods/extrahorns/init.lua]: Please don't align commas          , there should be no space between the comma and the previous word. You just fixed this in one craft recipe. The rest of the code still has these.

File: mods/extrahorns/init.lua [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+            gain              = 1,
+        })
+    end,
+})
+
+minetest.register_craft ({
+    type   = "shapeless",
+    output = "extrahorns:largehorn",
+    recipe = {"soundblocks:smallhorn", "default:steel_ingot"}
+
```
Comment (ClobberXD) [inline mods/extrahorns/init.lua]: Please remove this extra empty line.

File: mods/extrahorns/README.md (hunk 4 above)
Comment (ClobberXD) [reply mods/extrahorns/README.md]: Did you push the commit(s)? It doesn't seem to be fixed.

**Review by ClobberXD — COMMENTED** [review]
- The code is still mostly indented by spaces, and only some lines are tab-indented.
- Please delete the `sounds/original` directory
- The texture names are inconsistent. Make sure to follow the correct naming scheme for textures as given [here](https://github.com/minetest/minetest/blob/9854340c0b21d01a94ba7ef526c775e59e37b898/doc/lua_api.txt#L358). e.g. `"extrahorns_largehorn.png"`, `"extrahorns_warhorn.png"`, etc.

**Comment by ClobberXD** [comment]
The only thing that remains is to remove the `original` directory in `sounds`.

**Comment by DonutNinja** [comment]
Due to a messup on my fork, this pull request will be re opened - once you have read this, delete this pull request 
i just wanted to leave a message so you werent confused :D

**Pull request closed without merging**